README.md

```diff
@@ -56,5 +56,18 @@ Simple queueing service built for Internet Computer
 - Send & read messages simultaneously
 - Once processed only
 
+---------
 
+#### <b>DFX Commands for local testing</b>
 
+```
+$ dfx deploy --mode=reinstall --argument '(opt record {principal "<principal-id>"; null})' icsqs  
+```
+
+#### <b>To call methods:</b> dfx canister call <canister_id> <method_name> '(<arguments>)'
+examples:
+```
+$ dfx canister call <canister_id> sendMessage '("<message>")'
```

Comment:
same is it possible to automate these ?

Reply:
we can use it like this:
`for i in `seq 1 100`; do echo “dfx canister call rrkah-fqaaa-aaaaa-aaaaq-cai sendMessage ‘(\“TEST MESSAGE $i\“)‘“; done | bash -`
should i add this to readme?

Reply:
no need for loop, just one statement is fine